send_conn.go

```diff
@@ -42,9 +43,9 @@ func newSendConn(c rawConn, remote net.Addr, info packetInfo, logger utils.Logge
 	}
 
 	oob := info.OOB()
-	// add 32 bytes, so we can add the UDP_SEGMENT msg
+	// increase oob slice capacity, so we can add the UDP_SEGMENT and ECN control messages without allocating
 	l := len(oob)
-	oob = append(oob, make([]byte, 32)...)
+	oob = append(oob, make([]byte, 64)...)
 	oob = oob[:l]
```

Comment:
NIT: will 
```
oob = append(oob, make([]byte, 64)...)[:len(oob)] 
```
have the same effect? 
Feel free to ignore this.

Reply:
It does, I tried it out: https://go.dev/play/p/C83Gg3GKzju.
But it's not obvious too me that this should work. I prefer the slightly longer but clearer version. I'll move the `[:l]` one line up though.